FEM/Output.cpp

```diff
@@ -561,6 +567,24 @@ ios::pos_type COutput::Read(std::ifstream& in_str,
             _tecplot_zones_for_mg = true;
             continue;
         }
+        // for tecplot: Number output files for multiple output on same domain
+        if (line_string.find("$NUMBER_TEC_OUTPUT") != string::npos)
+        {
+            _number_outputs = true;
+            continue;
+        }
+        // For tecplot node-nr output JM
+        if (line_string.find("$NODE_INDEX_OUTPUT") != string::npos)
+        {
+            _node_output = true;
+            continue;  // JM
+        }
+        // For tecplot quadratic node output JM
+        if (line_string.find("$QUAD_NODE") != string::npos)
```

Comment:
minor comment: To me, the name "QUAD_NODE" is not clear about what it does. I would recommend QUAD_NODE_OUTPUT.

Reply:
Since it is a subkeyword of #OUTPUT, I don't know if this is necesary. However, I will rename it.